usaspending_api/disaster/v2/views/disaster_base.py

```diff
@@ -120,10 +120,12 @@ def all_closed_defc_submissions(self):
                 sub.fiscal_year == self.reporting_period_min_year
                 and sub.fiscal_period >= self.reporting_period_min_month
             ) or sub.fiscal_year > self.reporting_period_min_year:
-                q |= (
-                    Q(submission__reporting_fiscal_year=sub.fiscal_year)
-                    & Q(submission__quarter_format_flag=sub.is_quarter)
-                    & Q(submission__reporting_fiscal_period__lte=sub.fiscal_period)
+                q |= Q(
```

Comment:
Are these changes necessary? The way the function is written it will run multiple sub-queries, one for each fiscal year (2020 forward)